assets/model_monitoring/components/src/data_drift_compute_metrics/run.py

```diff
@@ -23,6 +23,8 @@ def run():
     parser.add_argument("--numerical_metric", type=str)
     parser.add_argument("--categorical_threshold", type=str)
     parser.add_argument("--numerical_threshold", type=str)
+    parser.add_argument("--override_numerical_feature", type=str)
```

Comment:
can we add nargs='+' to make it a list of str?

Reply:
It is a comma-joint string so we can just do a split(",")

Reply:
then please rename the parameter to --override_numerical_feature**s**